test/ColumnsPaginatedBookViewTests.ts

```diff
@@ -38,7 +38,7 @@ describe("ColumnsPaginatedBookView", () => {
         });
 
         it("should set iframe and iframe body width and height and column width based on window width and set height", () => {
-            (document.body as any).offsetWidth = 100;
+            (window as any).innerWidth = 100;
```

Comment:
I'm sure there's a good reason for it, but why are these casts of window to `any` needed? Just curious!